app/controllers/admin/email_controller.rb

```diff
@@ -157,7 +157,7 @@ def handle_mail
     retry_count = 0
 
     begin
-      Jobs.enqueue(:process_email, mail: email_raw, retry_on_rate_limit: true, source: :handle_mail)
+      Jobs.enqueue(:process_email, mail: email_raw, retry_on_rate_limit: true, source: "handle_mail")
```

Comment:
@davidtaylorhq blast from the past! This somehow still triggers warnings:

```
notice: Jobs::ProcessEmail was enqueued with argument values which do not cleanly serialize to/from JSON. This means that the job will be run with slightly different values than the ones supplied to `enqueue`. Argument values should be strings, booleans, numbers, or nil (or arrays/hashes of those value types). (deprecated since Discourse 2.9) (removal in Discourse 3.0) 
At /var/www/discourse/app/controllers/admin/email_controller.rb:160:in `handle_mail`
```

(on meta) 👀